docs/README.md

```diff
@@ -7,6 +7,7 @@
 - [Frequently Asked Questions](FAQ.md)
 - [Common Issues](Common-Issues.md)
 - [`brew` Shell Completion](Shell-Completion.md)
+- [Linuxbrew: Homebrew on Linux and Windows](Linuxbrew.md)
```

Comment:
There is an `Installation` page above for mac, should we add a comment there for a Linux installation? Or should we try to regroup both installation procedures in one page?

Reply:
Eventually I'd like the same installation procedure (one-line shell snippet) to be used for both macOS and Linux. In the mean time, https://linuxbrew.sh will be the go-to resource for installation instructions.

Reply:
"and Windows 10".